test/e2e/data/infrastructure-vsphere/kustomization/topology/kustomization.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - cluster-template-topology-base.yaml
+  - ../commons/cluster-resource-set.yaml
```

Comment:
Since the cluster-resource-set.yaml is being added as a resource, the file `cluster-template-topology-base.yaml` should not have those entries.